### PR TITLE
Learn revamp - what is ethereum page

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Revamp "What is Ethereum?" page.

https://ethereumorgwebsitedev01-whatisethrevamp.gtsb.io/en/what-is-ethereum/

## TODOs
- [x] Fetch infromation for the stats grid.
- [x] Display tooltip in stats grid.
- [x] Polish mobile version.
- [x] Add custom events tracking

### CODE REVIEW

File: src/pages-conditional/what-is-ethereum.tsx [hunk 1]
```diff
@@ -359,11 +388,13 @@ const WhatIsEthereumPage = ({
               <Translation id="page-what-is-ethereum-desc" />
             </Slogan>
             <Subtitle>
-              <Translation id="page-what-is-ethereum-accessibility" />
+              Learn more about how Ethereum works, the benefits it brings and
```
Comment (samajammin) [inline src/pages-conditional/what-is-ethereum.tsx:391]:
```suggestion
              Learn how Ethereum works, the benefits it brings and
```

Comment (samajammin) [reply src/pages-conditional/what-is-ethereum.tsx:391]: I'd suggest we rephrase this section to:

> A complete beginner's guide to how Ethereum works, the benefits it brings and how it is being used by millions of people around the world.

**Comment by samajammin** [comment]
Stats page doesn't look great on my 13inch Mackbook Air:
Image 2022-06-30 at 8 58 58 PM

Could we decrease the breakpoint here to keep on 3 per row?
Image 2022-06-30 at 9 00 31 PM

Also the `stat.formattedValue`  😆

File: src/components/Slider.tsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  flex-direction: row;
+  margin-left: 2rem;
+  margin-bottom: 2rem;
+`
+
+const Button = styled.button`
```
Comment (samajammin) [inline src/components/Slider.tsx:53]: Let's add a `cursor: pointer;` on these buttons.

**Comment by samajammin** [comment]
@pettinarip noticing the nav is broken on preview deploy? I'm seeing this in my local env as well:
Image 2022-07-05 at 4 38 20 PM

File: src/pages-conditional/what-is-ethereum.tsx [hunk 3]
```diff
@@ -374,141 +405,694 @@ const WhatIsEthereumPage = ({
...
+              </p>
+              <p>
+                These distributed nodes, run by individuals and businesses all
+                over the world, provide resiliency to the Ethereum network
+                infrastructure. It is therefore much less vulnerable to hacks or
+                shutdowns.{" "}
+                <b>
+                  Since its launch in 2015, Ethereum has never suffered downtime
+                </b>
+                . There are currently over 1777 individual nodes running
```
Comment (samajammin) [inline src/pages-conditional/what-is-ethereum.tsx:917]: This number (1777) should be pulled in dynamically via the Etherscan API.

Comment (pettinarip) [reply src/pages-conditional/what-is-ethereum.tsx]: We have changed the copy here. Updated

File: src/pages-conditional/what-is-ethereum.tsx [hunk 4]
```diff
@@ -374,141 +405,694 @@ const WhatIsEthereumPage = ({
...
-              description={card.description}
+      </Content>
+      <StyledGrayContainer>
+        <Section>
+          <TwoColumnContent id="summary">
+            <Width60>
+              <Summary>
+                <h2>Summary</h2>
+                <p>
+                  Ethereum is a technology that lets you send cryptocurrency to
```
[the remark this answers is not in the document]
Comment (pettinarip) [reply src/pages-conditional/what-is-ethereum.tsx]: The summary copy was updated but using the latest changes on Figma.

**Review by wackerow — APPROVED** [review]
What an awesome update, great work @pettinarip and everyone involved =)

**Comment by wackerow** [comment]
Passing build: https://www.gatsbyjs.com/dashboard/5e02f963-3fe5-4e37-a1e7-ac05f7753a86/sites/dc7c4340-a8e9-44ed-b7be-883c70d2f02d/builds/8fa194b7-4224-4aad-86f4-25a16798579e/details

**Comment by pettinarip** [comment]
> I don't see either of these as blockers but after the release we should fix:
> 
> Horizontal scroll caused by BannerGrid (320px-350px) tested on Brave latest.
> 
> Screenshot 2022-07-07 at 19 46 36
> 
> Squashed chart: (could we render fewer options in the chart at smaller screen sizes?)
> 
> Screenshot 2022-07-07 at 19 46 19
> 
> Otherwise, looks good to me. There are a few changes that @konoleoda and I discussed bringing in but those can wait.

Yes, on the chart I just did that. We are hiding the Gold and Gaming columns on small res.

**Pull request merged**